### PR TITLE
FIX: BIometrics unlock

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: UnlockWith.js [hunk 1]
```diff
@@ -121,9 +121,9 @@ const UnlockWith = () => {
       }
 
       setBiometricType(bt);
-      if (!biometricType || storageIsEncrypted) {
```
Comment (Overtorment) [inline UnlockWith.js:124]: its a shame that this wasnt caught when it was initially merged.
also, i suppose TS would have caught this automatically

Comment (Overtorment) [reply UnlockWith.js:124]: i checked, it was me who "reviewed" and  approved it: https://github.com/BlueWallet/BlueWallet/pull/6034

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ojgbf5kym7cczptmpk6b5hhgbi?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/335)

File: UnlockWith.js (hunk 1 above)
Comment (marcosrdz) [reply UnlockWith.js:124]: I doubt TS would've caught this as the value and type are valid. just the wrong item being checked against

**Pull request merged**